app/controllers/settings/labs.js

```diff
@@ -11,7 +11,7 @@ export default Controller.extend({
     submitting: false,
     showDeleteAllModal: false,
 
-    importMimeType: 'application/json',
+    importMimeType: ['application/json', 'application/zip'],
```

Comment:
It does, agreed 100%. Nothing there yet to test the existing file upload yet. Might take a bit longer, as I don't think it's possible in an acceptance test... 😔 
Maybe @acburdine has an idea for that?